Dockerfile

```diff
@@ -1,12 +1,28 @@
-FROM        registry.access.redhat.com/ubi8/nodejs-14
+FROM registry.access.redhat.com/ubi8/nodejs-16
 
-RUN         npm install -g yarn
+USER 0
 
-COPY        . .
+RUN set -eux && \
+  npm set progress=false && \
+  npm set update-notifier=false && \
+  npm set audit=false && \
+  npm set fund=false && \
+  npm install -g npm@latest
```

Comment:
Do we need to install `npm` again here? Or the one in the image is good enough?

Reply:
You don't like to keep dependencies up-to-date?

That said. Removed in https://github.com/app-sre/unleash/pull/42/commits/378f7a123ebfb1b48df12baee48d6dd60e1b585e.

Thank you.